apps/emqx_modules/src/emqx_recon.erl

```diff
@@ -32,6 +32,8 @@ enable() ->
 disable() ->
     emqx_ctl:unregister_command(recon).
 
+cmd(["observer_cli"]) ->
```

Comment:
How do stop?

Reply:
`ctrl+C` or enter `q` to exit.

Reply:
the command run as same as 
```erlang
emqx remote_console
> observer_cli:start().
```
the exit way is the same.